### ISSUE TITLE
WCAG Bug, Stufe A- Dem Inlineframe fehlt ein Alternativtext

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/rrze-video#251]
<img width="1993" height="823" alt="Image" src="https://github.com/user-attachments/assets/bd272b4b-489f-4898-b615-2ddadfaff839" />

https://my2.siteimprove.com/Inspector/1241073/A11Y/Page?pageId=142876361444&impmd=1A3574CC7C3B5ABC4A1CA70A1865D7A2&decision=dismissed+falsePositive+none#/sia-r13/failed
